### PR TITLE
cdk changes

### PR DESCRIPTION
**Pull request by renukakrishnan**
*Description of changes:*
CDK project for Codebuild 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: applications/dotnet/src/northwind-cdk/src/NorthwindCdk/NorthwindCdkStack.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                Environment = new BuildEnvironment
+                {
+                    BuildImage = LinuxBuildImage.AMAZON_LINUX_2_ARM_2,
+                    Privileged = true
+                },
+                EnvironmentVariables = new Dictionary<string, IBuildEnvironmentVariable>()
+                {
+                    { "AWS_REGION", new BuildEnvironmentVariable { Type = BuildEnvironmentVariableType.PLAINTEXT, Value = this.Region} },
+                    { "AWS_ACCOUNT_ID", new BuildEnvironmentVariable { Type = BuildEnvironmentVariableType.PLAINTEXT, Value = this.Account} },
+                    { "GITEA_URL", new BuildEnvironmentVariable { Type = BuildEnvironmentVariableType.PLAINTEXT, Value = "gitea.renukakn.people.aws.dev"} },
```
Comment (shapirov103) [inline applications/dotnet/src/northwind-cdk/src/NorthwindCdk/NorthwindCdkStack.cs:40]: the value won't work ootb, how did you plan passing it over? Maybe add a CLI command to kick off the build project overriding these env variables, we can then include as a manual step until we can make it git driven.

Comment (shapirov103) [reply applications/dotnet/src/northwind-cdk/src/NorthwindCdk/NorthwindCdkStack.cs:40]: Also @renukakrishnan can we set the name of the code build project to something meaningful? it is now is "NoSourceProject129711B8-ImCzB5S9q2EV" in my codebuild

Comment (renukakrishnan) [reply applications/dotnet/src/northwind-cdk/src/NorthwindCdk/NorthwindCdkStack.cs:40]: We can oveeride the env variables while calling the codebuild start command. If we implement the gitea webhook + lambda function, the lambda code will trigger the build passing the env variables.

Comment (shapirov103) [reply applications/dotnet/src/northwind-cdk/src/NorthwindCdk/NorthwindCdkStack.cs:40]: Example command in the README will help.

**Comment by shapirov103** [comment]
Closing as we have a separate CICD process.

**Pull request closed without merging**